RTProtocol.cs

```diff
@@ -423,8 +423,8 @@ public int ReceiveRTPacket(out PacketType packetType, bool skipEvents = true, in
                     while (receivedTotal < frameSize)
                     {
                         // As long as we haven't received enough data, wait for more
-                        received = mNetwork.Receive(ref data, receivedTotal, frameSize - receivedTotal, false, timeout);
-                        if (received <= 0)
+                        received = mNetwork.Receive(ref data, receivedTotal, frameSize - receivedTotal, false, -1);
+                        if (received < 0)
```

Comment:
Packets with size 0 are associated with a graceful close of a TCP connection
```suggestion
                        if (received <= 0)
```